src/node_crypto.h

```diff
@@ -320,6 +320,13 @@ class ByteSource {
   const char* get() const;
   size_t size() const;
 
+  inline operator bool() const {
```

Comment:
I'm not super fond of type coercion. You save a few keystrokes but introduce new opportunities for subtle bugs.